src/main/java/com/example/SS2_Backend/model/stableMatching/Individual.java

```diff
@@ -15,6 +15,19 @@
 import static com.example.SS2_Backend.util.Utils.isDouble;
 import static com.example.SS2_Backend.util.Utils.isInteger;
 
+/**
+ * A standard individual entity in a one-to-many matching problem model.
```

Comment:
Cho Stable Matching Problem chứ không riêng gì one-to-many nhé, sửa lại.

Reply:
:)